src/bson.rs

```diff
@@ -134,19 +134,38 @@ impl Display for Bson {
             Bson::Array(ref vec) => {
                 fmt.write_str("[")?;
 
+                let indent_str;
+                if let Some(width) = fmt.width() {
+                    indent_str = " ".repeat(width);
+                } else {
+                    indent_str = "".to_string();
+                }
```

Comment:
this can be made a bit more concise with:

```rust
let indent_str = if let Some(width) = fmt.width() {
    " ".repeat(width)
} else {
    "".to_string()
};
```

(ditto elsewhere with a similar pattern)

Reply:
I updated this to use unwrap_or to be more concise